orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/OperationsController.groovy

```diff
@@ -201,6 +202,13 @@ class OperationsController {
     return [message: e.message, status: HttpStatus.BAD_REQUEST]
   }
 
+  @ResponseStatus(HttpStatus.BAD_REQUEST)
+  @ExceptionHandler(PipelineValidator.PipelineValidationFailed)
+  Map disabledPipelineHandler(PipelineValidator.PipelineValidationFailed e) {
+    return [message: e.message, status: HttpStatus.BAD_REQUEST]
+  }
+
+  @Deprecated
   static class DisabledPipelineException extends RuntimeException {
```

Comment:
We can get rid of this previous exception when we remove v1. For now we might as well keep it. It only traps manually triggered executions.